### ISSUE TITLE
Supporter Badge Brightness

### ISSUE THREAD

**Issue by ultrawidegamer** [Yellow-Dog-Man/Resonite-Issues#2986]
### Describe the Underlying Issue

After switching the supporter badge to use a matcap, it still appears a bit too dark and dull but looks much more consistent overall.

### To Reproduce

Look at the supporter badge and compare it to the rest of the badges

### Expected Behavior

Supporter badge should have a similar brightness level to that of other badges

### Screenshots / Video

![2024-09-20 05 58 53](https://github.com/user-attachments/assets/bf87a474-0031-4965-ae19-f875998b6a13)

### Additional Context

I have attached a couple of new proposed matcaps to use 

First one where it is only slightly brighter than the original
![Matcap1](https://github.com/user-attachments/assets/26968f69-8cc0-4d67-bd32-e130ee2022c0)
![2024-09-20 05 59 07](https://github.com/user-attachments/assets/1025fa16-a442-403b-ba97-d2b0235ad722)

 
and second one which is my preferred one because it matches closer to that of the host star badge
![Matcap2](https://github.com/user-attachments/assets/15ba31e3-7272-4810-897e-b00c58976830)
![2024-09-20 05 59 17](https://github.com/user-attachments/assets/22c25bd6-78c0-40dc-8bd5-c033ebe4daa5)

### Reporters

ultrawidegamer
navy3001

**Comment by shiftyscales** [comment]
The prior reported feedback was concerns over the material's appearance drastically changing depending on lighting conditions in the world- that was resolved.

In my opinion, the examples you posted look significantly worse than how the badge currently looks, and looses its intended metallic / shiny appearance.

There are also other badges with a dark palette in the official badge lineup, so I don't think there's anything that need be done here.

![{9938F8F9-AA0A-4B23-B54B-A05ED0D80090}](https://github.com/user-attachments/assets/0a3d6098-e7a3-49b8-82c3-c3d06d0da08e)

**Comment by stiefeljackal** [comment]
> In my opinion, the examples you posted look significantly worse than how the badge currently looks, and looses its intended metallic / shiny appearance.

@shiftyscales Can you post examples from your end so that we can see the intended metallic, shiny appearance?

**Comment by TisFoolish** [comment]
I think what they mean is that Ultra's edits make the coin brighter, but less shiny. 

To me the current comes across as a burnished bronze with Ultra's edits coming across as a toony, fake gold? 

I think the current looks better than both what we had before and better than Ultra's edits, but i do wonder if the intent is supposed a burnished bronze. 

How would it look with a realistic gold matcap?

**Comment by shiftyscales** [comment]
@stiefeljackal - @ultrawidegamer did so- it's the first image in the post. I was referring to the edits following, yes.

**Comment by stiefeljackal** [comment]
> @stiefeljackal - @ultrawidegamer did so- it's the first image in the post. I was referring to the edits following, yes.

I see now. Thank you for the clarification.

**Comment by ultrawidegamer** [comment]
@shiftyscales My main concern is the inconsistency in the appearance of the badge. Most badges don't use a dark color palette, and the current badge looks drastically different from what’s shown in the original post and the sessions tab.

original post
![image](https://github.com/user-attachments/assets/6c1899e5-a96a-423b-af29-c27eb2a6fd8e)


sessions tab
![2024-09-15 18 13 24](https://github.com/user-attachments/assets/a03da3a3-2896-4e0a-b2e3-2987db18754c)


I’m not against the badge having a stylized, shiny look, but it should maintain a consistent design across all areas to avoid confusion about what the badge represents.

**Comment by shiftyscales** [comment]
Yeah- the original post was made when it used a metallic material in a brightly lit environment. As observed from prior feedback- it only ever looked like that at that specific angle in that specific lighting condition, which is why we switched to a matcap material per that feedback so the material would look the same under all lighting conditions.

Matcaps change appearance depending on which angle it is viewed from- but it remains self-consistent regardless of the lighting conditions because it's an unlit material.

https://github.com/user-attachments/assets/975b7bdd-c986-498c-aed8-8bcb75fa95dd

If you are concerned about it not having that design on the session menu- that'd require additional engineering time to display it properly with the matcap material in the session UI. Changing the badge's appearance in-world wouldn't resolve the issue you are raising.

It could be done- but would it be worth the engineering effort? It's still the same shape- is anyone actually getting confused over what the badge is- and would having it render the exact same way in the session UI actually resolve that?

Per my prior post- while darker colours on official badges are less common, they still exist in several badges. I don't believe it makes a lot of sense to clamp the range of possible designs and colours a badge can use to fulfill an arbitrary sense of 'uniformity'.

**Comment by ultrawidegamer** [comment]
As described by @TisFoolish, the color was looks to resemble 'burnished bronze.' 

Here’s an image of what 'burnished bronze' typically looks like in real life
![image](https://github.com/user-attachments/assets/069ec299-218e-4797-bb0a-ce3743154c3e)

And, giving the benefit of the doubt, let’s also compare it to brass
![image](https://github.com/user-attachments/assets/84132dcd-ad09-43ad-8c7d-b38950d80bad)

However, this is the badge as it appears in the sessions tab, the original version
![image](https://github.com/user-attachments/assets/69f39f10-1974-48cd-8395-e1a48a33c686)

And here’s how it looks above your head
![image](https://github.com/user-attachments/assets/fda41a2d-42c1-4f8b-afd8-ee672843c658)

If this is the final version you're going with and are happy, I'll have to happily decline using it on my avatar.
